slides/08-Platform-Hub-New/1050-ProjectsAndKubeTokens.md

```diff
@@ -0,0 +1,13 @@
+## Projects and Kube Tokens
+
+ - Projects in which you are a team member of can be found in the Projects tab within the Org section
```

Comment:
All projects are shown in the Projects section, not just the ones you are a member of.

Also, you use the word `tab` here, but `section` elsewhere. Probably best to stick to `section` consistently.

Reply:
so from what I can see, my use of the word `section` else where also has the same meaning. If you have an alternative for `tab` and `section` I'll use that